packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -218,7 +218,7 @@
       "properties": {
         "salesforcedx-vscode-apex-replay-debugger-checkpoints.enabled": {
```

Comment:
Won't it be better to just remove this whole properties? That's what we did before after we are ready to GA something that was guarded.

Or, do you 

1. Think that is too risky?
1. Envision that you might want to actually have some people be able to toggle between the two?

Reply:
@vazexqi we (the team) had talked about this in standup earlier this week. We decided that leaving the flag in, for the moment, was the way we wanted to handle this right now. We are ultimately going to remove this but with the current timeline this was the safest change.